apps/settings/locales/settings.en-US.properties

```diff
@@ -290,7 +290,12 @@ message-delivery-reports = Delivery Reports
 message-delivery-reports-details = Request notification of delivery for each message sent
 message-read-reports = Read Reports
 message-read-reports-details = Request read report for each message sent
-mmsTitle = Multimedia Messaging (MMS)
+# LOCALIZATION NOTE (mmsTitle): LRE (&#x202a;) and PDF (&#x202c;)
+# force the proper directionality of parentheses at the end of the string
+# regardless of the direction of the surrounding layout. Remove them only
+# if the translation puts them between other text.
+# mmsTitle.innerHTML= &#x202a;Multimedia Messaging (MMS)&#x202c;
```

Comment:
You can probably remove line 297 here as it is the same as the original string one line below.